src/main/java/Main.java

```diff
@@ -10,7 +12,17 @@ public static void main(String[] args) {
     }
 
     void run(Scanner scanner) {
-        // usupełnij metodę
+        CountryReader countryReader = new CountryReader();
+        HashMap<String, Country> countryMap = countryReader.createMap("countries.csv");
+        File file = new File("countries.csv");
+        if (file.exists()) {
+            System.out.println("Podaj kod kraju, o którym chcesz zobaczyć informacje:");
+            String countryCode = scanner.nextLine();
+            if (countryMap.get(countryCode) != null) {
+                System.out.println(countryMap.get(countryCode));
```

Comment:
W warunku if ładniej jest użyć metody `contains`

Jeżeli w kilku miejscach powtarza się wywołanie metody get() to warto stosować taki schemat:
```
Country country = countryMap.get(countryCode);
if (country != null) {
  sout(country);
} ...
```
(lepiej użyć jednak contains, bo kod po prostu lepiej się czyta jako "jeżeli kraj istnieje" niż "jeżeli kraj nie jest nullem").